core/src/test/java/com/amplifyframework/core/model/ModelSchemaTest.java

```diff
@@ -70,7 +70,7 @@ public void modelSchemaIsGeneratedForPersonModel() throws AmplifyException {
         expectedFields.put("dob", ModelField.builder()
             .targetType("AWSDate")
             .name("dob")
-            .type(Date.class)
+            .type(Class.forName("com.amplifyframework.core.model.temporal.Temporal$Date"))
```

Comment:
This is a bit of a hack because I cannot reference `Temporal.Date.class` from here, since it is in the `aws-api-appsync` package, so I have to get it using reflection.   A better solution for another day could be to move more of the `Model` classes from `core` to `aws-api-appsync`.   

Previously, this test was only passing by coincidence - it was comparing `java.lang.Date.class.getSimpleName()` with the `Temporal.Date.class.getSimpleName()`, and it passed since they are equal.  Now that we are comparing the actual class, this test started failing (as it should), so this needs to be updated to actually reference the correct `Class`.

Reply:
> aws-api-appsync module, which I can't reference from core

Certainly not from _src_ code. But what about as a `testImplementation`?

You could also access it through the `testutils` module, perhaps -- an additional level of indirection?